### DISCUSSION TITLE
Optical Flow Deck Parameter

### DISCUSSION THREAD

**Discussion by tharindusuraj** [bitcraze/discussions#376] [Q&A]
In the mm_flow.c code, what is the reason to consider the thetapix value as 4.2 degrees even though the actual FoV of the PMW3901 is 42 degrees?

**Comment by knmcguire** [comment]
Hi!

To this day this still has been a mystery... We have an [issue](https://github.com/bitcraze/crazyflie-firmware/issues/890) open for it and so far it has always worked, so it must be compensated for it in some other way as the estimation is not off by a factor of 10 degrees.

So yeah, we haven't gotten around to properly investigate it!
